src/components/DisplayUnitTag.tsx

```diff
@@ -0,0 +1,48 @@
+import { useCallback } from 'react';
+
+import _ from 'lodash';
+import { useDispatch } from 'react-redux';
+
+import { DisplayUnit } from '@/constants/trade';
+
+import { useAppSelector } from '@/state/appTypes';
+import { setDisplayUnit } from '@/state/configs';
+import { getSelectedDisplayUnit } from '@/state/configsSelectors';
+
+import { Tag } from './Tag';
+
+type ElementProps = {
+  assetId?: string;
+  className?: string;
+  entryPoint: string;
+};
+
+export const DisplayUnitTag = ({ entryPoint, assetId, className }: ElementProps) => {
+  const displayUnit = useAppSelector(getSelectedDisplayUnit);
+  const dispatch = useDispatch();
+
+  const dispatchSetDisplayUnit = _.debounce((newDisplayUnit) => {
+    if (!assetId) return;
+    dispatch(setDisplayUnit({ newDisplayUnit, entryPoint, assetId }));
+  }, 300);
```

Comment:
interesting - why do we debounce here? also what do ya think of making a constants/debounce file? (I see 50, 200, 1000 used across product and it might be nice to just consolidate to a few values)